### PR TITLE
Mention metrics comparison on experiment tracking documentation

### PR DESCRIPTION
**Pull request by AhdraMeraliQB**
## Description
This addition was made because we haven't told users that they can compare metrics over time in the metadata sidep-anel.

## Checklist

- [x] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [x] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro/pull/1707"></a>

### CODE REVIEW

File: docs/source/tutorial/set_up_experiment_tracking.md [hunk 1]
```diff
@@ -174,4 +174,8 @@ You can now access, compare and pin your runs by toggling the `Compare runs` but
 
 ![](../meta/images/experiment-tracking_demo.gif)
 
+Additionally, you can also monitor the changes to metrics over time from the pipeline visualisation tab ![](../meta/images/pipeline_visualisation_icon.png). Clicking on any [MetricsDataset](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.tracking.MetricsDataSet.html)  node will open a sidepanel displaying how the metric value has changed over time.
```
Comment (yetudada) [inline docs/source/tutorial/set_up_experiment_tracking.md:177]:
```suggestion
Additionally, you can also monitor the changes to metrics over time from the pipeline visualisation tab ![](../meta/images/pipeline_visualisation_icon.png). Clicking on any [MetricsDataset](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.tracking.MetricsDataSet.html) node will open a side panel displaying how the metric value has changed over time.
```

**Review by yetudada — APPROVED** [review]
This is awesome! Thank you @AhdraMeraliQB!

File: docs/source/tutorial/set_up_experiment_tracking.md [hunk 2]
```diff
@@ -174,4 +174,8 @@ You can now access, compare and pin your runs by toggling the `Compare runs` but
 
 ![](../meta/images/experiment-tracking_demo.gif)
 
+Additionally, you can also monitor the changes to metrics over time from the pipeline visualisation tab ![](../meta/images/pipeline_visualisation_icon.png). Clicking on any [MetricsDataset](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.tracking.MetricsDataSet.html) node will open a side panel displaying how the metric value has changed over time.
```
Comment (MerelTheisenQB) [inline docs/source/tutorial/set_up_experiment_tracking.md:177]: Additionally and also have pretty much the same meaning so using one is enough.

```suggestion
Additionally, you can monitor the changes to metrics over time from the pipeline visualisation tab ![](../meta/images/pipeline_visualisation_icon.png). Clicking on any [MetricsDataset](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.tracking.MetricsDataSet.html) node will open a side panel displaying how the metric value has changed over time.
```

**Review by MerelTheisenQB — APPROVED** [review]
Great to have the gif to show how to find the metrics over time!

**Review by SajidAlamQB — APPROVED** [review]
This looks great, the only suggestion may be to record a higher resolution gif as its kind of hard to see.

**Comment by AhdraMeraliQB** [comment]
> This looks great, the only suggestion may be to record a higher resolution gif as its kind of hard to see.

@SajidAlamQB I think it's due to how the screen recordings are compressed; tried some workarounds but couldn't see much improvement so leaving as is.

**Pull request merged**